packages/sw-routing/build.js

```diff
@@ -15,38 +15,20 @@
 
 const commonjs = require('rollup-plugin-commonjs');
 const nodeResolve = require('rollup-plugin-node-resolve');
-const path = require('path');
 const pkg = require('./package.json');
-const {buildJSBundle} = require('../../build-utils');
+const {buildJSBundle, generateBuildConfigs} = require('../../build-utils');
 
-const pluginsConfig = [
+const plugins = [
```

Comment:
There's no transpilation needed. The only thing we're transpiling elsewhere is `async`/`await` ➡️  generators, and there's none of that in this bundle.